apps/bbsync/tests/test_query.py

```diff
@@ -870,7 +870,7 @@ def test_unembargo(self):
         )
 
         new_flaw = Flaw.objects.first()
-        new_flaw.embargoed = False
+        new_flaw.acl_read = []  # make it whatever but embargoed
```

Comment:
What about `acl_write`? :thinking:

Reply:
The `is_embargoed` property really cares only about `acl_read` so for the sake of the test `acl_write` is not important.